app/_landing_pages/dev-portal/self-service.yaml

```diff
@@ -0,0 +1,213 @@
+metadata:
+  title: "Developer self-service"
+  content_type: landing_page
+  description: "Enable self-service registration flows for developers and applications using authentication strategies and {{site.konnect_short_name}} application auth."
+  products:
+    - dev-portal
+  tags:
+    - application-registration
+    - authentication
+  breadcrumbs:
+    - /dev-portal/
+
+rows:
+
+  - header:
+      type: h1
+      text: "Developer self-service and app registration"
+      sub_text: Enable self-service registration flows for developers and applications using authentication strategies and {{site.konnect_short_name}} application auth.
+
+  - columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    {{site.konnect_short_name}} Dev Portal provides flexible options for controlling access to content and APIs. 
+                    When combined with a [Gateway Service](/gateway/entities/service/), developers visiting a Dev Portal can sign up, create an application, register it with an API, and retrieve API keys without intervention from Dev Portal administrators. 
+
+                    Developer self-service consists of two main components:
+                    * **User authentication:** Allows users to access your Dev Portal by logging in. You can further customize what logged in users can see using RBAC.
+                    * **Application registration:** Allows developers to use your APIs using credentials and create applications for them.
+
+                    To enable developer self-service, do the following:
```

Comment:
Let's put a heading here (eg Enable developer self-service), otherwise the steps blend into the rest of the page.